pom.xml

```diff
@@ -330,6 +330,7 @@
     <module>sdk-workflows</module>
     <module>sdk-springboot</module>
     <module>examples</module>
+    <module>testcontainers-dapr</module>
```

Comment:
Please, follow naming convention from other projects.

Reply:
@artursouza we are following the conventions from testcontainers modules. Which reminds me that we need to update this when this PR is merged: https://testcontainers.com/modules/dapr/

Reply:
We need to discuss this to understand which one is more important to users.

Reply:
Let's keep the testcontainers naming convention here. Just add a comment next to it, so we all remember that in the future and someone else does not try to rename it.

Reply:
done